src/lua/fiber.c

```diff
@@ -740,10 +754,15 @@ lbox_fiber_serialize(struct lua_State *L)
 static int
 lbox_fiber_tostring(struct lua_State *L)
 {
-	char buf[32];
-	struct fiber *f = lbox_checkfiber(L, 1);
-	snprintf(buf, sizeof(buf), "fiber: %llu",
-		 (long long)f->fid);
+	char buf[35];
+	uint64_t fid = luaT_fiber_id(L, 1);
+
+	if (fiber_find(fid) != NULL) {
+		snprintf(buf, sizeof(buf), "fiber: %llu", (long long)fid);
+	} else {
+		snprintf(buf, sizeof(buf), "fiber %llu is dead",
+			 (long long)fid);
```

Comment:
Nit: Let's change to `"fiber: %llu (dead)"` for consistency with the format used for alive fibers.

Also, let's use `lua_pushfstring` instead of formatting the string to a buffer on stack.

Reply:
Changed the string.

`lua_pushfstring` cannot be used, because it doesn't support the 64-bit uint format specifier.